common/utils.js

```diff
@@ -33,13 +32,37 @@ export function getUniqueString (prefix = DEFAULT_PREFIX) {
  */
 export function getRandomElement (array, seed) {
   if (seed) {
-    const hash = fnv.hash(seed);
-    return array[hash.value % array.length];
+    const hash = javaHashCode(seed);
+    return array[Math.abs(hash) % array.length];
   } else {
```

Comment:
Is this function only used for Avatar or are there any other components that make use of it? If not, maybe we can rename it to `getAvatarBackgroundColor` or something more specific?

Reply:
The idea was it could be used for any array I don't think it needs to be specific to avatar.

Reply:
Awesome, having one algo for deciding coloring would be great!